control_panel_api/settings/base.py

```diff
@@ -132,7 +132,7 @@
 LOGS_BUCKET_NAME = os.environ.get('LOGS_BUCKET_NAME', 'moj-analytics-s3-logs')
 IAM_ARN_BASE = os.environ.get('IAM_ARN_BASE', '')
 K8S_WORKER_ROLE_NAME = os.environ.get('K8S_WORKER_ROLE_NAME', '')
-SAML_PROVIDER_ARN = os.environ.get('SAML_PROVIDER_ARN', '')
+SAML_PROVIDER = os.environ.get('SAML_PROVIDER', '')
```

Comment:
As [said in twin PR](https://github.com/ministryofjustice/analytics-platform-helm-charts/pull/43/files#r146847965), I wonder if we should be a bit more explicit and call it `SAML_PROVIDER_NAME`.

Reply:
(FYI: I added another comment in the other PR, probably this setting name is fine)